File: ScoreSystem/ScoreSystem/ScoreSystem/Model/StudentScoreTrendResult.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ScoreSystem.Model
{
    public class StudentScoreTrendResult
    {
        private string studentName;
        private string studentNumber;
        private Dictionary<string, List<ScoreTrendPoint>> trend;

        public string StudentName { get => studentName; set => studentName = value; }
        public string StudentNumber { get => studentNumber; set => studentNumber = value; }
        public Dictionary<string, List<ScoreTrendPoint>> Trend { get => trend; set => trend = value; }
    }
}

File: ScoreSystemServer/ScoreSystemServer/src/main/java/com/score/system/entity/trend/StudentScoreTrendResult.java
package com.score.system.entity.trend;

import lombok.Data;

import java.util.List;
import java.util.Map;

@Data
public class StudentScoreTrendResult {
    private String studentName;
    private String studentNumber;
    private Map<String,List<ScoreTrendPoint>> trend;
}
